### PR TITLE
Treat 403s from Europeana as dead links

### LINKED ISSUE #2417: Treat a 403 response code from Europeana as dead link
## Description

Today we found a lot of broken images from Europeana returning `403 Forbidden` on direct and foreign landing URLs. To fix it we need to include an entry for Europeana in this dictionary:

https://github.com/WordPress/openverse/blob/6da2118ade7acff7035abb0df42e17578e8ecab3/api/api/utils/check_dead_links/provider_status_mappings.py#L13-L23

## Reproduction

Look at the results from https://api.openverse.engineering/v1/images/?source=europeana&q=Arco%20de%20flecha e.g.
	- https://api.openverse.engineering/v1/images/27d7186f-2f8e-4d24-badd-d6e4bdc60071/ or
	- https://api.openverse.engineering/v1/images/5dee57d2-7547-4b33-964e-f0ca8e5196d4/

## Additional context

Similar issues with other providers: #1201 #940.

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Fixes #2417 by @krysal

## Description

Adds the exception for Europeana as said in the issue.

## Testing Instructions

1. To manually test these changes, copy this row to the **sample_data/sample_image.csv** file
```
27d7186f-2f8e-4d24-badd-d6e4bdc60071,2022-11-27 14:37:37.668997+00,2022-11-27 14:37:37.668997+00,provider_api,europeana,europeana,/2022709/oai_fototeca_mcu_es_fototeca_MORENO_04766_C,https://www.mcu.es/fototeca_patrimonio/show_ficha.do?archivo=MORENO&record=04766_C,https://www.mcu.es/fototeca_patrimonio/Visor?usarVisorMCU=false&archivo=MORENO/preview/04766_C_P.jpg,,,,,by-nc-nd,4.0,,,Arco de flechas,"{""license_url"": ""https://creativecommons.org/licenses/by-nc-nd/4.0/"", ""raw_license_url"": null}",,f,2022-11-27 14:37:37.668997+00,f,jpg,,
```
2. Confirm you have the environment variable `FILTER_DEAD_LINKS_BY_DEFAULT` set to `True` you don't have it set to anything (it's true by default)
3. Run `just api/recreate`
4. Confirm the following search returns zero result http://localhost:50280/v1/images/?source=europeana

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: api/env.docker [hunk 1]
```diff
@@ -19,5 +19,3 @@ SEMANTIC_VERSION=1.0.0
 ELASTICSEARCH_URL=es
 
 WATERMARK_ENABLED=True
-
```
Comment (obulat) [inline api/env.docker:22]: Why do we remove variable this from here?

**Review by obulat — APPROVED** [review]

File: api/env.docker (hunk 1 above)
Comment (krysal) [reply api/env.docker:22]: We should strive to remove this file entirely. I was confused at first when my local API wasn't filtering dead links given I had the variable commented in my `.env` file, then I found this was the problem. These are environment settings so it's not a good practice to track them in vcs either.

Comment (sarayourfriend) [reply api/env.docker:22]: Now that `env.template` isn't used for production (was it ever?) and neither is `env.docker`, and #2180 is implemented, I don't think these `env.docker` files make sense at all, really. We have appropriate defaults in `env.template` for local development, which is the only place that file is ever used, so this file feels duplicative indeed.

Comment (sarayourfriend) [reply api/env.docker:22]: I opened an issue to track this and fix it generally: https://github.com/WordPress/openverse/issues/2437

**Review by sarayourfriend — APPROVED** [review]
LGTM!

**Pull request merged**